backend/Makefile

```
@@ -0,0 +1,11 @@
+#---- COMMANDS ----#
+
+start: ##@Docker (start)
+	docker compose up --build
+
+fresh: ##@Docker (start) with fresh fixtures
+	docker compose --profile fresh down
+	docker compose --profile fresh up --build
+
+down: ##@Docker (stop) Stops and removes all containers
+	docker compose --profile fresh down
```

Clean is handy to have in debugging:

```suggestion
down: ##@Docker (stop) Stops and removes all containers
	docker compose --profile fresh down

clean: ##@Docker (clean) Delete all unused Docker containers, images and volumes
	docker compose down -v
	docker system prune --volumes
```

---

Is the description for clean right? And is clean the right name? As it only affects the volumes?
Also, the fixture dockerfile wipes all collections before importing the fixtures, so is it really necessary for the project?